book/src/Setup.md

```diff
@@ -224,6 +224,20 @@ by default for alpha):
 --feature-gates=VolumeSnapshotDataSource=true
 ```
 
+## Topology (alpha)
+In order to support topology-aware dynamic provisioning mechanisms available in Kubernetes, the *external-provisioner* must have the Topology feature enabled:
+
+```
+--feature-gates=Topology=true
```

Comment:
Im testing topology support using EBS CSI driver, doesn't seem this feature gate is required for me. And this flag is not present in [kube_features.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go) too. Is this a real feature gate?

Reply:
This is a provisioner flag